packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2075,7 +2126,8 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
   /// themselves as repaint boundaries must not replace the layer created by
   /// the framework.
   ///
-  /// Warning: This getter must not change value over the lifetime of this object.
+  /// If the value of this getter changes, [markNeedsCompositingBitsUpdate] must
```

Comment:
The comment above about the framework creating the OffsetLayer isn't 100% correct anymore (it should at least tell you that you can change that behavior now.

Reply:
done